context/app/static/js/components/home/EntityCount/EntityCount.jsx

```diff
@@ -1,22 +1,25 @@
 import React from 'react';
 import Typography from '@material-ui/core/Typography';
 import Skeleton from '@material-ui/lab/Skeleton';
+import { SecondaryBackgroundTooltip } from 'js/shared-styles/tooltips';
 
 import { FlexLink, StyledDiv } from './style';
 
 function EntityCount({ icon, count, label, href }) {
   return (
-    <FlexLink href={href}>
-      <StyledDiv>{icon}</StyledDiv>
-      <div>
-        <Typography variant="h2" component="p">
-          {count || <Skeleton />}
-        </Typography>
-        <Typography variant="h6" component="p">
-          {label}
-        </Typography>
-      </div>
-    </FlexLink>
+    <SecondaryBackgroundTooltip title={`${count} ${label}`} placement="bottom-end">
+      <FlexLink href={href}>
+        <StyledDiv>{icon}</StyledDiv>
+        <div>
+          <Typography variant="h2" component="p">
+            {count > 9999 ? `${(count / 1000).toFixed(1)}k+` : count || <Skeleton />}
```

Comment:
It could be beneficial to pull the logic into a util and add a simple unit test. It would make the line more readable as well.

Reply:
Done!